source/extensions/transport_sockets/tls/io_handle_bio.cc

```diff
@@ -0,0 +1,130 @@
+#include "envoy/buffer/buffer.h"
+#include "envoy/network/io_handle.h"
+
+#include "openssl/bio.h"
+#include "openssl/err.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace TransportSockets {
+namespace Tls {
+
+namespace {
+
+static inline Envoy::Network::IoHandle* bio_io_handle(BIO* bio) {
```

Comment:
static doesn't do anything in unnamed namespace :)

Reply:
Thanks for that! Wanted to maintain the "typical" BIO formatting but even clang_tidy is complaining now. Is there any macro/comment that we could use to disable clang_tidy for this file? Do we want that or should I rename everything?

Reply:
I'm fine with keeping the naming, but not `static`. You can add NOLINT or NOLINTNEXTLINE for each method.

Reply:
Thanks and done!